app/src/components/Form/DemoValidationFormFlow.js

```diff
@@ -0,0 +1,123 @@
+
+// NOTE: this is a demo validation form flow - might be helpful in some validation case
```

Comment:
Delete this file if it's not needed.

Reply:
I will leave this file because it is a demo validation for them. I would like to use it for validating form field but I have no time to refactor the FE code yet. I'll probably pass to them to do it.